README.md

```diff
@@ -2091,7 +2100,7 @@ Other Style Guides
         let x = 1;
         break;
       case 2:
-        const y = 2;
+        const x = 2; // SyntaxError: Identifier 'x' has already been declared
```

Comment:
Otherwise example doesn't make sense. Variables should have the same names in both cases